### PR TITLE
feat(minutes): adding minutes from f2f meeting on 2019-10-10

### PR DESCRIPTION
**Pull request by mcjaeger**
Please feedback.

from recent face to face meeting on Oct 10th in Darmstadt at ESA find:

* meeting minutes
* group picture
* partial slides

Meeting monutes

Signed-off-by: Michael C. Jaeger <michael.c.jaeger@siemens.com>

### CODE REVIEW

File: Tooling-Landscape/Meeting-Material/Meeting-20191010/OSTG 20191011 tooling meeting minutes.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+(raw notes from notable items from discussion)
+
+## Introduction
+
+* 0915 general introduction to the location
+* 0920 Starting introduction of every participant
+
+## Updates on Tools
+
+* 0940 Integration demo: SW360antenna and SW360
```
Comment (blaumeiser-at-bosch) [inline Tooling-Landscape/Meeting-Material/Meeting-20191010/OSTG 20191011 tooling meeting minutes.md]: Suggestion O.R.T with Maven analyzer, SW360antenna and SW360

File: Tooling-Landscape/Meeting-Material/Meeting-20191010/OSTG 20191011 tooling meeting minutes.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+* 1010 Integration demo: Conan, O.R.T. and SW360
```
Comment (blaumeiser-at-bosch) [inline Tooling-Landscape/Meeting-Material/Meeting-20191010/OSTG 20191011 tooling meeting minutes.md]: Suggestion O.R.T. with Conan analyzer, SW360antenna and SW360

Comment (kallesoranko) [reply Tooling-Landscape/Meeting-Material/Meeting-20191010/OSTG 20191011 tooling meeting minutes.md]: @blaumeiser-at-bosch that's a good idea

**Comment by blaumeiser-at-bosch** [comment]
@mcjaeger , Michael, could you just make the two proposed changes and then we can merge this one as well.

File: Tooling-Landscape/Meeting-Material/Meeting-20191010/OSTG 20191011 tooling meeting minutes.md (hunk 2 above)
Comment (mcjaeger) [reply Tooling-Landscape/Meeting-Material/Meeting-20191010/OSTG 20191011 tooling meeting minutes.md]: done.

File: Tooling-Landscape/Meeting-Material/Meeting-20191010/OSTG 20191011 tooling meeting minutes.md (hunk 1 above)
Comment (mcjaeger) [reply Tooling-Landscape/Meeting-Material/Meeting-20191010/OSTG 20191011 tooling meeting minutes.md]: done.

**Pull request merged**